### DISCUSSION TITLE
Sharing My Experience with iNav 7.1.2 VTOL Automated Switching

### DISCUSSION THREAD

**Discussion by ivolanski** [iNavFlight/inav#10550] [General]
Hi all,

I wanted to share my experience using iNav 7.1.2 with my Hee Wing Ranger T1 VTOL aircraft. During the setup process, I found the information on the [iNav VTOL documentation](https://github.com/iNavFlight/inav/blob/master/docs/VTOL.md) incredibly helpful. A big thank you to everyone involved in creating and maintaining such great resources!

While going through the configuration, one detail caught my attention: the section about Automated Switching, which mentioned that it is "one of the least tested features." Despite this note, I decided to give it a try, and I’ve been thoroughly impressed.

To date, I’ve tested the automated switching function more than 100 times, and it has performed flawlessly every single time. The seamless transitions between modes have added an extra layer of confidence and enjoyment to my flights.

I want to take this opportunity to thank the entire iNav team for their incredible work. Your dedication has made such a powerful and reliable system available to hobbyists like me. I hope this feedback helps contribute to the continued improvement of iNav.

Keep up the fantastic work—I love what you’ve created and look forward to seeing how iNav continues to evolve!

**Comment by kasatka60** [comment]
Can you share the settings for automatic switching? Have you checked the automatic landing? And is there a video demonstration?
I have T1 VTOL.

**Comment by ivolanski** [comment]
Hi, automatic switching settings are in the link above, named as inav documentation. Yes I am doing auto landing after automatic switching and it is working great, just set RTH to land.
